### DISCUSSION TITLE
Refreshview pull to refresh horizontally

### DISCUSSION THREAD

**Discussion by FM1973** [dotnet/maui#10365] [Q&A]
Hey there.

I got a collectionview inside a refreshview.
The collectionview has it´s itemslayout set like this:

`<GridItemsLayout Orientation="Horizontal" HorizontalItemSpacing="5" />`

Is there a way to force the refreshview to "refresh" pulling right instead of down?
Pull down doesn´t work in this case anyway...

Thanks for your help!

**Comment by xtuzy** [comment]
You mean collectionview will update after pull right, or current item will update? maybe you can add a gesture for they and will show a ActivityIndicator. 
Another, I feel this is not a nice design.

**Reply by FM1973** [reply]
Good idea... thanks. I don´t like it as a design as well, but it´s a customer requirement...
